### DISCUSSION TITLE
聊天

### DISCUSSION THREAD

**Discussion by lingbopro** [lingbopro/soberjs-vscode#3] [General]
每日废话杂谈(1/1)

**Comment by Minemetero** [comment]
话说我Table-Tennis-Counter的Bug你得修啊

**Reply by lingbopro** [reply]
emmm

会修的

**Reply by Minemetero** [reply]
oh

**Comment by Minemetero** [comment]
![image_2024-07-04_163719814](https://github.com/lingbopro/soberjs-vscode/assets/156386015/2745f9e8-43d8-4d54-a1f8-93876a7c46d8)
这都谁下的。。。。

**Reply by lingbopro** [reply]
一个是我，一个是 @H3gg387 罢

剩下的我就不到了

**Comment by lingbopro** [comment]
![Screenshot_20240705_093748.jpg](https://github.com/lingbopro/soberjs-vscode/assets/148617407/4815a60f-f5d7-4908-aac3-9b1a4b044f93)

现在已经被装了6次了…

**Reply by Minemetero** [reply]
e谁装的呢。。。这个半成品。。。

**Reply by Minemetero** [reply]
CSS的提示不显示我很生气（

**Comment by Minemetero** [comment]
新版本多了个picker...

**Reply by lingbopro** [reply]
又来活了

**Reply by lingbopro** [reply]
所以Table-Tennis-Counter的sober到底咋回事

**Reply by Minemetero** [reply]
我不知道 版本冲突？而且他破坏性升级了

**Reply by Minemetero** [reply]
所以引入方式变了

**Reply by Minemetero** [reply]
你帮我搞一下网页我帮你搞这个扩展）

**Reply by lingbopro** [reply]
现在Table-Tennis-Counter正常了

**Reply by lingbopro** [reply]
> 你帮我搞一下网页我帮你搞这个扩展）

em

**Comment by Minemetero** [comment]
![image_2024-07-06_193010636](https://github.com/lingbopro/soberjs-vscode/assets/156386015/307f96ea-8627-4b7c-a128-d61aa97f2df6)
不会用就别用。。。。

**Reply by lingbopro** [reply]
um?

**Reply by Minemetero** [reply]
Token 这东西确实难搞

**Reply by lingbopro** [reply]
这个东西我本地测试没问题，扔上去就不行

我估计是Token的问题
Token错误就会这样

**Comment by Minemetero** [comment]
![image_2024-07-06_202344505](https://github.com/lingbopro/soberjs-vscode/assets/156386015/8849f4a3-a39a-4773-952c-392053a9591e)
这什么东西

**Reply by Minemetero** [reply]
这Valueset为啥在外面啊

**Reply by lingbopro** [reply]
> 这Valueset为啥在外面啊

`ValueSet`定义一个全局的枚举集

**Comment by Minemetero** [comment]
我不理解为什么你不把所有组件都给加进来。。。。。

**Comment by Minemetero** [comment]
![image_2024-07-10_103708746](https://github.com/lingbopro/soberjs-vscode/assets/156386015/0acf2d2a-0db3-4a67-a2b5-80974d3bb4d2)
你还有X和facebook？

**Reply by lingbopro** [reply]
这三个按钮的意思是分享到X，Facebook，和邮箱:)

**Reply by Minemetero** [reply]
Oh我还以为是你的X，Facebook

**Comment by lingbopro** [comment]
![image](https://github.com/lingbopro/soberjs-vscode/assets/148617407/c13ed0f6-794a-42f8-b244-7e79e1a8dfb2)

12个人装...

**Reply by lingbopro** [reply]
![image](https://github.com/lingbopro/soberjs-vscode/assets/148617407/69f6b554-7da1-4f00-8ef1-b5bae301ea8c)

但是VSCode显示只有7个

**Reply by Minemetero** [reply]
额

**Comment by Minemetero** [comment]
![image_2024-07-11_121005991](https://github.com/lingbopro/soberjs-vscode/assets/156386015/7ff6ad6b-c623-4d67-b2ae-1e2cacf7a7bd)
我们被官方star了 yeah!

**Reply by lingbopro** [reply]
ohhhh

**Reply by lingbopro** [reply]
yes

**Comment by Minemetero** [comment]
我觉得我们可以给apprat提个issue,建个Studio集中维护sober相关的东西

**Reply by lingbopro** [reply]
studio?

**Reply by Minemetero** [reply]
工作室

**Comment by Minemetero** [comment]
![image_2024-07-27_203448133](https://github.com/user-attachments/assets/21b316aa-734a-4b2f-b34f-60311aea1058)
很好。。。

**Reply by lingbopro** [reply]
脚本是这样的（

**Comment by lingbopro** [comment]
新版本我push到dev分支了

**Comment by Minemetero** [comment]
![image_2024-10-07_133655974](https://github.com/user-attachments/assets/e5d03da3-115a-436d-a4d7-5af01e243bd5)
6

**Reply by lingbopro** [reply]
因为dev还没合并进master嘛

**Reply by Minemetero** [reply]
为什么你今天不用上学

**Reply by lingbopro** [reply]
> 为什么你今天不用上学

昂？今天是国庆放假a

**Reply by Minemetero** [reply]
ಥ_ಥ

**Comment by Minemetero** [comment]
我有个主意那个STJ的网页我们改一改，摇身一变就是SoberJS的模板了

**Reply by Minemetero** [reply]
搞个Soberjs Community

**Reply by Minemetero** [reply]
把我们的Wiki，模板，还有这个扩展放到Soberjs Community下面

**Reply by lingbopro** [reply]
有道理（）

我一直不知道官网上的那个”模板“是干什么的

**Reply by Minemetero** [reply]
me2 lol
